modules/Contact.js

```diff
@@ -0,0 +1,16 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export default () => {
+  const timeContact = document.querySelector('#time');
+  const now = DateTime.now().setZone('Europe/London');
+  const dateTime = now.toLocaleString({
+    month: 'long',
+    day: 'numeric',
+    year: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+    hour12: true,
+  });
+  timeContact.textContent = dateTime;
+};
```

Comment:
- [ ] The date and time correctly show in my own end. Shows in UTC + 1 :heavy_check_mark: 

![image](https://user-images.githubusercontent.com/86318284/235447056-9ddfe770-3b3e-4580-82c7-a85816049f38.png)

- [ ] [OPTIONAL] It would be a good thing to name your files intuitively. The contents of this file have nothing to do with "contact"

Reply:
Thank you @Nemwel-Boniface for the thorough review. Yes, it was a misunderstanding on my side in terms of the file names, I fixed that. 😊